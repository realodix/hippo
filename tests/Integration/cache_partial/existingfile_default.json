{
    "fixer": {
        "tests\/Integration\/tmp\/existingfile_default_actual.txt": {
            "file_hash": "c98da5519f49b822",
            "blocks": [
                "cb0fb9d5fa164897",
                "a30ec6eb5f246b48",
                "fd94107060f4417e",
                "4dfd0946f2c12e71"
            ]
        }
    }
}
