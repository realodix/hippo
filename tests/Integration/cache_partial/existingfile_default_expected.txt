a
b
c
e
f
zz
g
h
i
j
